docs/UserGuide.md

```diff
@@ -145,17 +145,23 @@ Examples:
 
 #### Listing residents by fet/collection deadlines : `deadline`
 
-Lists residents whose ART collection or FET tests are due before the given date or the within the range of the 2 given dates.
+Lists residents whose ART collection or FET tests are either:
+* due before a given date, `d1`, by using the `late keywords` or 
+* due within the range of two given dates, `d1` and `d2`, by using the `normal keywords`.
+
+There is a one week deadline for the test kit collection and fet test, therefore due date refers to one week after the last recorded date
 
 Format: `deadline k/KEYWORD d1/DATE1 d2/DATE2` or `deadline k/LATE_KEYWORD d1/DATE1`
 
+* A resident's fet or collection is due one week after their last fet date or last collection date. For example, if a resident's last fet date is on a friday, `15-10-2021`, then the resident's fet deadline is on the following friday which is `22-10-2021`
+* The number of days a resident is considered late is calculated from `a day after` their deadline to the `current date`, both inclusive
```

Comment:
Can add an example here for more clarity

Reply:
Good idea